File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/zig/ADV-01_R.zig
const std = @import("std");

pub fn main() !void {
    var t = std.Io.Threaded.init_single_threaded;
    const io = t.io();
    const out = std.Io.File.stdout();
    try out.writeStreamingAll(io, "ADV-START\n");

    var ib: [64]u8 = undefined;
    var r = std.Io.File.stdin().readerStreaming(io, &ib);
    const a: i64 = try std.fmt.parseInt(i64, try r.interface.takeSentinel('\n'), 10);

    const b: i32 = @intCast(a);

    var ob: [64]u8 = undefined;
    try out.writeStreamingAll(io, try std.fmt.bufPrint(&ob, "OBS=V:{d}\n", .{b}));
    try out.writeStreamingAll(io, "ADV-END\n");
}

File: benchmark/template/programs/go/adversarial/ADV-01_R.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

func main() {
	fmt.Println("ADV-START")
	sc := bufio.NewScanner(os.Stdin)
	sc.Scan()
	a, _ := strconv.ParseInt(sc.Text(), 10, 64)
	v := int32(a)
	fmt.Printf("OBS=V:%d\n", v)
	fmt.Println("ADV-END")
}
